mobile-widgets/qmlmanager.cpp

```diff
@@ -144,15 +144,16 @@ QMLManager::QMLManager() : m_locationServiceEnabled(false),
 	timer.start();
 	connect(qobject_cast<QApplication *>(QApplication::instance()), &QApplication::applicationStateChanged, this, &QMLManager::applicationStateChanged);
 
-	QString libdcLogFileName = QStandardPaths::standardLocations(QStandardPaths::GenericDataLocation).first() + "/libdivecomputer.log";
-	logfile_name = copy_qstring(libdcLogFileName);
 #if defined(Q_OS_ANDROID) || defined(Q_OS_IOS)
 #if defined(Q_OS_ANDROID)
 	appLogFileName = QStandardPaths::standardLocations(QStandardPaths::GenericDataLocation).first() + "/subsurface.log";
+	QString libdcLogFileName = QStandardPaths::standardLocations(QStandardPaths::GenericDataLocation).first() + "/libdivecomputer.log";
 #elif defined(Q_OS_IOS)
 	// on iOS we should save the data to the DocumentsLocation so it becomes accessible to the user
 	appLogFileName = QStandardPaths::standardLocations(QStandardPaths::DocumentsLocation).first() + "/subsurface.log";
+	QString libdcLogFileName = QStandardPaths::standardLocations(QStandardPaths::DocumentsLocation).first() + "/libdivecomputer.log";
 #endif
+	logfile_name = copy_qstring(libdcLogFileName);
```

Comment:
Since this is now only done for Android or iOS, Subsurface-mobile on desktop might end up behaving in uncool ways, right? There's no longer a filename set for it - and no way to set it.

Reply:
Actually no, the filename is libdivecomputer.log, and there was no way before to change the name.

I have on purpose not removed the files, so both libdivecomputer.log and subsurface.log are still produced.

Reply:
Ah, so it just gets overwritten for iOS/Android, but otherwise it still works. Cool - I was afraid that it would be undefined in Subsurface-mobile with your change when compiling on Linux/Mac

Reply:
As far as I can see the same happens for the desktop version, but I might be mistaken (have not tested it, just looked at where the open statement is).

I am doing a lot to ensure that none of my patches break on Linux/Mac, because theoretically mobile could grow into something useful on these platforms as well :-)